shell/platform/common/cpp/client_wrapper/plugin_registrar.cc

```diff
@@ -138,12 +138,52 @@ void BinaryMessengerImpl::SetMessageHandler(const std::string& channel,
                                      ForwardToHandler, message_handler);
 }
 
+// Wrapper around a FlutterDesktopTextureRegistrarRef that implements the
+// TextureRegistrar API.
+class TextureRegistrarImpl : public TextureRegistrar {
+ public:
+  explicit TextureRegistrarImpl(
+      FlutterDesktopTextureRegistrarRef texture_registrar_ref)
+      : texture_registrar_ref_(texture_registrar_ref) {}
+
+  virtual ~TextureRegistrarImpl() = default;
+
+  // Prevent copying.
+  TextureRegistrarImpl(TextureRegistrarImpl const&) = delete;
+  TextureRegistrarImpl& operator=(TextureRegistrarImpl const&) = delete;
+
+  virtual int64_t RegisterTexture(Texture* texture) override {
+    FlutterTexutreCallback callback =
+        [](size_t width, size_t height, void* user_data) -> const PixelBuffer* {
+      return ((Texture*)user_data)->CopyPixelBuffer(width, height);
+    };
+    int64_t texture_id = FlutterDesktopRegisterExternalTexture(
```

Comment:
Since Windows is no longer using the GLFW embedding, but does use common/cpp/client_wrapper, you'll need to add stub implementations of the new functions to the Windows embedding as well. You can just have them log error messages indicating that texture support isn't implemented yet for Windows and pointing to https://github.com/flutter/flutter/issues/38601.